controllers/controllers.go

```diff
@@ -65,6 +62,12 @@ func AddToManager(ctx *context.ControllerManagerContext, mgr manager.Manager) er
 			return errors.Wrap(err, "failed to initialize VirtualMachinePublishRequest controller")
 		}
 	} else {
+		// We only update TKG related ContentSource/ContentLibraryProvider/ContentSourceBinding resources
+		// in provider configmap reconcile. These resources will be removed when the FSS is enabled,
+		// add provider configmap controller to the manager only when the FSS is disabled.
+		if err := providerconfigmap.AddToManager(ctx, mgr); err != nil {
```

Comment:
Nit: keep the controllers in alphabetical order, even in the conditional statement ;)